openstates/wy/bills.py

```diff
@@ -165,7 +160,7 @@ def scrape_digest(self, bill):
                     actor = 'upper'
 
                 date = datetime.datetime.strptime(date, '%m/%d/%Y')
-                bill.add_action(actor, action, date,
+                bill.add_action(actor, action.strip(), date,
```

Comment:
at some point we should pull out entities from this:

e.g. `"action": "Introduced and Referred to H07 - Corporations",` => `Corperations`, (although, that wouldn't resolve right :) )

Doesn't need to change right now, I'll file a `DATA-` ticket